certbot-apache/certbot_apache/assertions.py

```diff
@@ -51,10 +55,59 @@ def assertEqualDirective(first, second):
     assert isinstance(second, interfaces.DirectiveNode)
     _assertEqualDirectiveComponents(first, second)
 
+def assertEqualBlock(first, second):
+    """ Equality assertion for BlockNode """
+
+    # first was checked in the assertEqual method
+    assert isinstance(first, interfaces.BlockNode)
+    assert isinstance(second, interfaces.BlockNode)
+    _assertEqualDirectiveComponents(first, second)
+    # Children cannot be asserted, because Augeas implementation will not
+    # prepopulate the sequence of children.
+    # assert len(first.children) == len(second.children)
+
 def isPass(value): # pragma: no cover
     """Checks if the value is set to PASS"""
     return PASS in value
 
+def isPassDirective(block):
+    """ Checks if BlockNode or DirectiveNode should pass the assertion """
+
+    if block.name == PASS:
```

Comment:
We have `isPass`, might as well use it for these checks (ie `isPass(block.name)`, `isPass(block.parameters)`).